sshd-core/src/main/java/org/apache/sshd/server/config/keys/DefaultAuthorizedKeysAuthenticator.java

```diff
@@ -112,6 +112,11 @@ protected boolean isValidUsername(String username, ServerSession session) {
             if (violation != null) {
                 throw new IOException("String permission violation (" + violation + ") for " + path);
             }
+
+            String ownerViolation = KeyUtils.validateStrictKeyFileOwner(path);
+            if (ownerViolation != null) {
+                throw new IOException("String owner violation (" + ownerViolation + ") for " + path);
+            }
```

Comment:
I also wondered why there is a string, please push a fix to master and I will rebase.